modules/Header/ui/Header.tsx

```diff
@@ -218,3 +264,8 @@ export function Header({
         </header>
     );
 }
+
+function humanizeUrl(url: URL) {
+    const string = url.hostname.replace('www.', '');
```

Comment:
What if the domain name has `www` in it? Like `holycowww.com`? :) 

```suggestion
    const string = url.hostname.replace(/^www\./, '');
```

Reply:
You're right, I apologize for all the cowwws out there, thanks! 😆